core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -3716,7 +3716,9 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                           value: readv * 0 + sval */
                         DEF_ATOMIC_RMW_OPCODE(XCHG, *0 +);
                 }
-
+#if WASM_ENABLE_THREAD_MGR != 0
```

Comment:
isn't that a problem for the fast interpreter as well?

Reply:
of course, I forgot